docs/Architecture.md

```diff
@@ -18,7 +18,7 @@ architecture is oriented around a single central *Database*.
 ### Worker
 
 A Zonemaster *Worker* is a daemon that picks up *test* requests from the
-*Database*, runs them using the *Engine* library, and records the results back
+*Database*, runs them using the *Zonemaster Engine* library, and records the results back
 to the *Database*. A single *Worker* may handle several requests concurrently.
 The *Backend* architecture supports a single *Worker* interacting with a single *Database*.
```

Comment:
I think this last one should be kept as Zonemaster Engine because the Worker itself is the Test Agent.